management/conduct/telegram.md

```diff
@@ -10,11 +10,16 @@ Regolamento
 
 - [Sì] discussione su news, lavoro e lifestyle del web-developer e affini
 
+- [Sì] utilizzare paste tools, come pastebin.com o simili, per condividere il proprio codice
```

Comment:
È l'inverso della linea 17, l'obiettivo dovrebbe essere quello di ridurre al minimo il testo in modo da invogliare la lettura.